modules/wayfire/hm.nix

```
@@ -0,0 +1,64 @@
+{
+  config,
+  lib,
+  pkgs,
+  ...
+}:
+
+{
+  options.stylix.targets.wayfire.enable =
+    config.lib.stylix.mkEnableTarget "wayfire" true;
+
+  config =
+    lib.mkIf (config.stylix.enable && config.stylix.targets.wayfire.enable)
+      (
+        let
+          inherit (config.lib.stylix) colors;
+          rgb = rgb: ''\#${rgb}ff'';
+
+          wayfireCfg = config.wayland.windowManager.wayfire;
+
+          wayfireBackground = toString (
```

Maybe a more generic name might be better:

```suggestion
          background = toString (
```